scalyr_agent/builtin_monitors/kubernetes_monitor.py

```diff
@@ -1054,6 +1055,38 @@ def _prepare_to_stop(self):
         finally:
             self.__condition_var.release()
 
+    def mark_not_new(self, container_id):
+        """
+        Marks a WarmingEntry as no longer new
+        @param container_id: the container id of the warming entry to mark as not new
+        """
+        self.__lock.acquire()
+        try:
+            entry = self.__active_pods.get(container_id, None)
+            if entry:
+                entry.is_new = False
+
+        finally:
+            self.__lock.release()
+
+    def is_new(self, container_id):
```

Comment:
Likewise, this turns into `has_been_used`.

Reply:
As per conversation earlier, this was changed to `pending_first_use`.